package.json

```diff
@@ -38,6 +38,7 @@
     "font-awesome": "^4.6.1",
     "jquery": "~2.1.4",
     "origin-web-common": "~0.0.12",
+    "hopscotch": "~0.2.7",
```

Comment:
Not for this PR, but at some point we should discuss why/if we need to define dependencies in both NPM and Bower.